src/utils/helpers.ts

```diff
@@ -0,0 +1,3 @@
+// is this folder only supposed to be for scss helpers?
```

Comment:
Nope, it's fine to have javascript too. Maybe we can organize this folder `utils/styles` and `utils/scripts` ? It depends if we want to put all utilities scripts in the same file or not.

Reply:
As there's only 1 file of utility scripts, for now, I'll just keep this one helpers.ts file in the folder. Once we add more, we can break them into those folders.